cfme/control/explorer/alert_profiles.py

```diff
@@ -308,4 +308,4 @@ class VMInstanceAlertProfile(BaseAlertProfile):
 
 class NodeAlertProfile(BaseAlertProfile):
 
-    TYPE = "Node"
+    TYPE = version.pick({version.LOWEST: "Node", "5.9.2": "Container Node"})
```

Comment:
**required** this version pick cant happen at code loading time wit no appliance given
this code needs to be transformed into something that has appliance availiable

Reply:
Well theoretically you right, but I don't see how it possible to create this object without an appliance object.

You can see the way I used to create this kind of object, and as you can see it requires an appliance object

``` python
alert_profiles = appliance.collections.alert_profiles
node_profile = alert_profiles.instantiate(NodeAlertProfile, "Prometheus node Profile")
```
If you know another valid way to create this object, I will be happy to hear how you suggest solve this issue you described.

Reply:
as far as i can tell, the NodeAlertProfile object is used in navigation, as such it should awlays be a object

in that case, the appliance is availiable via the entity behaviour, it should be possible to make TYPE a lazy version pick that uses the appliance, i recall we have properties for that

Reply:
Possible workaround:
```python
class NodeAlertProfile(BaseAlertProfile):

    @property
    def TYPE(self):  # noqa
        if self.appliance.version < "5.9.2":
            return "Node"
        else:
            return "Container Node"
```

Reply:
@quarckster This workaround does not solve @RonnyPfannschmidt 's issue, you still need to have an appliance object

Reply:
@shalomnaim1 Every instance of `BaseEntity` class has `appliance` attribute which refers to current appliance object.

Reply:
@shalomnaim1 `version.pick` is not a lazy version pick.

Reply:
@mfalesni I do not understand what is the added value of "quick" version pick here?

Reply:
@shalomnaim1 I've improved `deferred_verpick` in #7002. You can use it in your code.